tests/queries/models.py

```diff
@@ -704,3 +704,20 @@ class RelatedIndividual(models.Model):
 
     class Meta:
         db_table = 'RelatedIndividual'
+
+
+class Ticket28297Recipe(models.Model):
```

Comment:
There's no need to put the ticket number in the model name. If possible to reuse any existing models, that's ideal.